targets/TARGET_ARM_SSG/mbed_rtx.h

```diff
@@ -22,9 +22,6 @@
 #ifndef INITIAL_SP
 #define INITIAL_SP              (0x20020000UL)
 #endif
-#ifndef OS_TASKCNT
-#define OS_TASKCNT              7
-#endif
 #ifndef OS_MAINSTKSIZE
 #define OS_MAINSTKSIZE          112
```

Comment:
Can OS_MAINSTKSIZE also be removed now?

What about initial_sp - surely we have this from the linker script or unified boot code now?